src/Services/YouTubeService.php

```
@@ -369,14 +387,15 @@ private function extractTagsFromCaption(string $caption): array
      * @param string $url The URL.
      * @throws SocialMediaException
      */
-    private function validateInput(string $caption, string $url): void
+    private function validateInput(string $caption, string $urlOrPath): void
```

PHPStan (2.2.8)

[error] PHPDoc tag `@param` references unknown parameter: $url


(parameter.notFound)